Core/clim-basic/extended-streams/recording.lisp

```diff
@@ -1392,12 +1410,12 @@ were added."
 
 (defrecord-predicate draw-polygon-output-record (closed filled)
   (and (if-supplied (closed)
-         (eql (slot-value record 'closed) closed))
+                    (eql (slot-value record 'closed) closed))
```

Comment:
maybe we should introduce a dummy macro if-supplied (or rather, when supplied!) so the indentation is right and that arguments are hinted in the modeline. This indentation does not match the macro.

Reply:
Indentation is fixed. I would prefer to do further improvements in a separate pull request.